numtoeng.py

```diff
@@ -0,0 +1,31 @@
+#As taught in class, using arrays and simple math to solve the problem
+#Time Complexity :O(n)
+#Space complexity: O(3n)
```

Comment:
Time & Space here is both O(1).
2^32 number needs only 10 iterations, in the worst case.